pkg/cluster/config/types.go

```diff
@@ -43,6 +43,12 @@ type Node struct {
 	// Role defines the role of the nodw in the in the Kubernetes cluster managed by `kind`
 	// Defaults to "control-plane"
 	Role NodeRole
+	// CNI is the name of the desired network plugin
+	CNI string
+	// CNIConfigFIleName is the name of the config file that will be placed in /etc/cni/net.d
```

Comment:
typo in comment
probably might want to omit the path and say e.g. "CNI plugin path".